rlp/sedes/big_endian_int.py

```diff
@@ -25,7 +25,7 @@ def serialize(self, obj):
             raise SerializationError("Can only serialize integers", obj)
         if self.length is not None and obj >= 256**self.length:
             raise SerializationError(
-                "Integer too large (does not fit in {} " "bytes)".format(self.length),
+                "Integer too large (does not fit in {self.length} bytes)",
```

Comment:
Missing `f` here. This will print "self.length" not the value of `self.length`

Reply:
Good catch, thank you!